docs/configuration/outbox-event-router.asciidoc

```diff
@@ -0,0 +1,195 @@
+
+[source,javascript,indent=0]
+----
+# Kafka Topic: outbox.event.order
+# Kafka Message key: "1"
+# Kafka Message Headers: "id=4d47e190-0402-4048-bc2c-89dd54343cdc"
+# Kafka Message Timestamp: 1556890294484
+{
+  "eventType": "OrderCreated",
+  "payload": "{\"id\": 1, \"lineItems\": [{\"id\": 1, \"item\": \"Debezium in Action\", \"status\": \"ENTERED\", \"quantity\": 2, \"totalPrice\": 39.98}, {\"id\": 2, \"item\": \"Debezium for Dummies\", \"status\": \"ENTERED\", \"quantity\": 1, \"totalPrice\": 29.99}], \"orderDate\": \"2019-01-31T12:13:01\", \"customerId\": 123}"
+}
+----
+
+This message is generated by transforming a Debezium raw message, which looks like:
+
+[source,javascript,indent=0]
+----
+# Kafka Message key: "406c07f3-26f0-4eea-a50c-109940064b8f"
+# Kafka Message Headers: ""
+# Kafka Message Timestamp: 1556890294484
+{
+  "before": null,
+  "after": {
+    "id": "406c07f3-26f0-4eea-a50c-109940064b8f",
+    "aggregateid": "1",
+    "aggregatetype": "Order",
+    "payload": "{\"id\": 1, \"lineItems\": [{\"id\": 1, \"item\": \"Debezium in Action\", \"status\": \"ENTERED\", \"quantity\": 2, \"totalPrice\": 39.98}, {\"id\": 2, \"item\": \"Debezium for Dummies\", \"status\": \"ENTERED\", \"quantity\": 1, \"totalPrice\": 29.99}], \"orderDate\": \"2019-01-31T12:13:01\", \"customerId\": 123}",
+    "timestamp": 1556890294344,
+    "type": "OrderCreated"
+  },
+  "source": {
+    "version": "0.9.3.Final",
+    "connector": "postgresql",
+    "name": "dbserver1-bare",
+    "db": "orderdb",
+    "ts_usec": 1556890294448870,
+    "txId": 584,
+    "lsn": 24064704,
+    "schema": "inventory",
+    "table": "outboxevent",
+    "snapshot": false,
+    "last_snapshot_record": null,
+    "xmin": null
+  },
+  "op": "c",
+  "ts_ms": 1556890294484
+}
+----
+
+This result was achieved with the link:#default-table-columns[Default Configuration] which assumes a table structure and event routing based on aggregates, in case you want a custom behavior, the smt is fully configurable, check the link:#configuration-options[Available configuration options].
+
+== Configuration
+
+=== Configuration options
+[cols="30%a,10%a,10%a,50%a",width=100,options="header,footer",role="table table-bordered table-striped"]
+|=======================
+|Property
+|Default
+|Group
+|Description
+
+|`table.field.event.id`
+|`id`
+|Table
+|The column which contains the Event ID within the outbox table
+
+|`table.field.event.key`
+|`aggregateid`
+|Table
+|The column which contains the Event Key within the outbox table, when this is set the value of this column will be used as a kafka message key
+
+|`table.field.event.type`
+|`type`
+|Table
+|The column which contains the Event Type within the outbox table
+
+|`table.field.event.timestamp`
+|
+|Table
+|Optionally you can override the Kafka message timestamp with a value from a chosen field, otherwise it'll be the debezium event processed timestamp.
```

Comment:
```suggestion
|Optionally you can override the Kafka message timestamp with a value from a chosen field, otherwise it'll be the Debezium event processed timestamp.
```